multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt

```diff
@@ -252,7 +252,7 @@ class MadaraGenerator : ThemeSourceGenerator {
         SingleLang("MangaSco", "https://manhwasco.net", "en", overrideVersionCode = 2),
         SingleLang("MangaSiro", "https://mangasiro.com", "en", isNsfw = true),
         SingleLang("MangaSpark", "https://mangaspark.com", "ar", overrideVersionCode = 2),
-        SingleLang("MangaStic", "https://mangastic.me", "en", overrideVersionCode = 1),
+        SingleLang("MangaStic", "https://mangastic9.com/", "en", overrideVersionCode = 2),
```

Comment:
```suggestion
        SingleLang("MangaStic", "https://mangastic9.com", "en", overrideVersionCode = 2),
```

Remove trailing slash.

Reply:
thanks, missed that :)